service/src/main/java/greencity/service/notification/NotificationServiceImpl.java

```diff
@@ -99,18 +102,38 @@ private boolean checkIfUnpaidOrderNeedsNewNotification(Order order, Optional<Use
                 || order.getOrderDate().isEqual(LocalDateTime.now(clock).minusDays(3)));
     }
 
-    private UserNotification initialiseNotificationForUnpaidOrder(Order order, UserNotification userNotification) {
-        userNotification.setNotificationTime(LocalDateTime.now(clock));
-        userNotification.setNotificationType(NotificationType.UNPAID_ORDER);
-        userNotification.setOrder(order);
-        UserNotification notification = userNotificationRepository.save(userNotification);
-        NotificationParameter notificationParameter = new NotificationParameter();
-        notificationParameter.setKey(ORDER_NUMBER_KEY);
-        notificationParameter.setValue(order.getId().toString());
-        notificationParameter.setUserNotification(notification);
-        NotificationParameter createdParameter = notificationParameterRepository.save(notificationParameter);
-        notification.setParameters(Collections.singleton(createdParameter));
-        return notification;
+    private Set<NotificationParameter> initialiseNotificationParametersForUnpaidOrder(Order order, Long amountToPay) {
+        Set<NotificationParameter> parameters = new HashSet<>();
+
+        parameters.add(NotificationParameter.builder().key(AMOUNT_TO_PAY_KEY)
+            .value(String.format("%.2f", (double) amountToPay)).build());
+
+        parameters.add(NotificationParameter.builder()
+            .key(ORDER_NUMBER_KEY)
+            .value(order.getId().toString())
+            .build());
+
+        final String testGreenCity = "https://greencity-ubs.testgreencity.ga/ubs/details-for-existing-order/"
+            + order.getId();
+        final String pickUpCity = "https://greencity-ubs.pick-up.city/ubs/details-for-existing-order/"
+            + order.getId();
+
+        String hostName = inetAddressProvider.getInetAddressHostName();
+
+        if ("www.testgreencity.ga".equals(hostName)) {
```

Comment:
can we store this data in the configs and load different data depending on the environment?